packages/turf-bezier/index.js

```diff
@@ -33,9 +33,9 @@ var Spline = require('./spline.js');
  * };
  *
  * var curved = turf.bezier(line);
+ * curved.properties = { stroke: '#0f0' };
  *
  * //addToMap
- * curved.properties = { stroke: '#0f0' };
  * var addToMap = [line, curved]
  */
```

Comment:
@DenisCarriere @rowanwins ok the code after `addToMap = ...` is not rendered in the map on the website. So what's the purpose of adding properties in the example, which highlight the result on the map, if they are not actually used for the map?